views/random-beers.hbs

```diff
@@ -0,0 +1,5 @@
+<div class = "cardholder">
```

Comment:
Some margin could be applied on the sides so the beer is rendered more centered.